exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -96,17 +101,30 @@ def _translate_key_values(key: Text, value: Any) -> KeyValue:
         any_value = AnyValue(double_value=value)
 
     elif isinstance(value, Sequence):
-        any_value = AnyValue(array_value=value)
+        any_value = AnyValue(
+            array_value=ArrayValue(values=[_translate_value(v) for v in value])
+        )
 
+    # Tracing specs currently does not support Mapping type attributes
     elif isinstance(value, Mapping):
```

Comment:
We should have an issue for this in the spec. Translating a map to a list of KeyValues should be defined in the spec, otherwise there's no guarantee of what other languages will do.

Reply:
Removing this logic branch since mapping types cannot be added as attributes currently, as defined in the [tracing spec](https://github.com/open-telemetry/opentelemetry-specification/blob/4a05eea66d1055db4db15e5f7cf35d74c55c1a6a/specification/common/common.md#attributes)